File: contest/a_loot_sharing/sol/sol.php
<?php


$line = trim(fgets(STDIN));
$b = intval($line);
$line = trim(fgets(STDIN));
$t = intval($line);
$line = trim(fgets(STDIN));
$n = intval($line);



if(2*$b+3*$t <= $n){
    echo "LOOT!" . "\n";
}else{
    echo "RHUM!" . "\n";
}

File: contest/a_loot_sharing/sol/sol.rs
use std::io;

fn main() {
    let b = read_int();
    let t = read_int();
    let n = read_int();

    if 2 * b + 3 * t <= n {
        println!("LOOT!");
    } else {
        println!("RHUM!");
    }
}

fn read_int() -> i32 {
    let mut line = String::new();
    io::stdin().read_line(&mut line).unwrap();
    line.trim().parse().unwrap()
}
